lib/capybara_accessible_selectors/helpers.rb

```diff
@@ -38,7 +38,7 @@ def within_fieldset(xpath, fieldsets)
         if current_xpath.is_a? XPath::Union
           current_xpath.expressions.map do |x|
             fieldset.descendant(x)
-          end.reduce(:+)
+          end.reduce(:+) # rubocop:disable Performance/Sum
```

Comment:
As an alternative to disabling the rule, would [Enumerable#sum](https://ruby-doc.org/3.2.2/Enumerable.html#method-i-sum) work here?

```ruby
current_xpath.expressions.sum { |x| fieldset.descendant(x) }
```

Reply:
It didn't like sum.  It is the xpath dsl and as sum starts with a default (i.e. it is `reduce(0, :+)`) and there is no null  expression for sum to start with.

However, it turned out `:+` was a alias for `:union` in xpath DSL, so I changed it to that which gets rid of the rubocop disable.